server/routes/appointments/create-and-edit/handlers/dateAndTime.ts

```diff
@@ -24,12 +25,14 @@ export class DateAndTime {
 
   @Expose()
   @Type(() => SimpleTime)
+  @ValidateIf(o => o.appointmentJourney.mode !== AppointmentJourneyMode.COPY)
```

Comment:
User cannot enter times when duplicating an appointment as they are copied from the original appointment